channels/urbdrc/client/libusb/libusb_udevman.c

```diff
@@ -667,56 +598,62 @@ static void udevman_load_interface(UDEVMAN* udevman)
 	udevman->iface.initialize = udevman_initialize;
 }
 
-static BOOL urbdrc_udevman_register_devices(UDEVMAN* udevman, const char* devices)
+static BOOL udevman_parse_device_id_addr(char** str, UINT16* id1, UINT16* id2, UINT16 max,
```

Comment:
It might be a nice addition to make this `const char** str` to indicate that we don´t modify the content.